modules/configuration/jsx/DiagnosisEvolution.js

```diff
@@ -0,0 +1,386 @@
+                                value={trajectoryData.Name}
+                                required={true}
+                            />
+                            <SearchableDropdown
+                                name='ProjectID'
+                                label='Project'
+                                options={this.state.formData.projects}
+                                onUserInput={this.setFormData}
+                                value={trajectoryData.ProjectID}
+                                required={true}
+                            />
+                            <SearchableDropdown
+                                name='visitLabel'
+                                label='Visit'
+                                options={this.state.formData.visits}
+                                onUserInput={this.setFormData}
+                                value={trajectoryData.visitLabel}
+                                required={true}
+                            />
+                            <SearchableDropdown
+                                name='instrumentName'
+                                label='Instrument'
+                                options={this.state.formData.instruments}
+                                onUserInput={this.setFormData}
+                                value={trajectoryData.instrumentName}
+                                required={true}
+                            />
+                            <TagsElement
+                                name='sourceField'
+                                id={dxEvolutionID}
+                                label='Source Field'
+                                options={this.state.formData.sourceFields}
+                                useSearch={true}
+                                strictSearch={true}
+                                onUserInput={this.setFormData}
+                                value={trajectoryData.pendingSourceField ?
+                                    trajectoryData.pendingSourceField :
+                                    null}
+                                items={trajectoryData.sourceField || []}
+                                required={true}
+                                btnLabel='Add Field'
+                                pendingValKey='pendingSourceField'
+                                onUserAdd={this.addSourceField}
+                                onUserRemove={this.removeSourceField}
+                            />
+                            <NumericElement
+                                name='orderNumber'
+                                min={1}
+                                max={100}
+                                label='Order Number'
+                                onUserInput={this.setFormData}
+                                value={trajectoryData.orderNumber}
+                                required={true}
+                            />
+                            <div>
```

Comment:
```suggestion
                            <div
                              style={{
                                display: 'flex',
                                justifyContent: 'center',
                              }}
                            >
```

this will align the Save and Reset buttons in a row

Reply:
done!